#9: Adds keybind support.

marklabrecque opened the pull request:
Happy to make any changes necessary here. First time writing LUA specifically, so any feedback is welcome :)

I tested these changes against my copy of the add-on distributed by Curseforge, but cloning the repo into my WoW folder sent me a bunch of errors - not sure what's up with that.

Thanks!

cybertk commented on Bindings.xml:
```diff
@@ -0,0 +1,5 @@
+<Bindings>
+    <Binding name="WEEKLY REWARDS" category="WeeklyRewards">
+        TOGGLE_WEEKLY_REWARDS_WINDOW()
```
It would be better if you could have the following changes:  
1. Expand the `Main:ToggleWindow()` here, then we don't need to introduce the new global var `TOGGLE_WEEKLY_REWARDS_WINDOW`
2. Replace `name="WEEKLY REWARDS"` to `name="WeeklyRewards"`, then you can reuse the predefined `_G["BINDING_NAME_WeeklyRewards"]` and the keybinding UI will show: **"Show/Hide the window"**

cybertk replied:
here you go:

```xml
<Bindings>
    <Binding name="WeeklyRewards" category="WeeklyRewards">
        WeeklyRewards:ExecuteChatCommands()
    </Binding>
</Bindings>
```

marklabrecque commented:
Thanks for the feedback @cybertk - I think I have addressed your concerns. I have tested these changes and they do work for me.

Let me know if there are any other concerns before merging.
